src/shared/utilities/cliUtils.ts

```diff
@@ -74,11 +74,7 @@ export const AWS_CLIS: { [cli in AwsClis]: Cli } = {
  * @param confirmBefore Prompt before starting install?
  * @returns CLI Path
  */
-export async function installCli(
-    cli: AwsClis,
-    confirm: boolean,
-    window: Window = Window.vscode()
-): Promise<string | undefined> {
```

Comment:
Still thinking if just returning `undefined` is the best way to represent cancel or explicitly throwing an error is the best. An error would be better if we had more cross-cutting error handling but right now we don't. The user 'cancelling' a workflow is definitely exceptional as far as the logic is concerned.